Tests/UITests/UITests.xcodeproj/project.pbxproj

```diff
@@ -634,15 +634,15 @@
 			repositoryURL = "https://github.com/StanfordSpezi/XCTestExtensions";
 			requirement = {
 				kind = upToNextMinorVersion;
-				minimumVersion = 0.4.1;
+				minimumVersion = 0.4.6;
 			};
 		};
-		2F85828029E7784C0021D637 /* XCRemoteSwiftPackageReference "XCTHealthKit" */ = {
+		97A7D7162A54D69700D85923 /* XCRemoteSwiftPackageReference "XCTHealthKit" */ = {
 			isa = XCRemoteSwiftPackageReference;
-			repositoryURL = "https://github.com/StanfordBDHG/XCTHealthKit";
+			repositoryURL = "https://github.com/philippzagar/XCTHealthKit";
 			requirement = {
-				kind = upToNextMinorVersion;
-				minimumVersion = 0.3.3;
+				branch = main;
+				kind = branch;
```

Comment:
Short comment to remember that we chance that back to the upstream version before we tag the release.